sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Diagnostics/BlobEventStoreEventSource.cs

```diff
@@ -392,7 +392,7 @@ public virtual void GetCheckpointComplete(string fullyQualifiedNamespace,
                                                   string consumerGroup,
                                                   string partitionId,
                                                   string clientIdentifier,
-                                                  DateTimeOffset lastModified)
+                                                  string lastModified)
```

Comment:
Yay, ETW?

Reply:
This fixes that outstanding AOT warning in Event Hubs